lib/tasks/report_primary_organisation.rake

```diff
@@ -0,0 +1,79 @@
+namespace :report_primary_organisation  do
+  desc "A one-off CSV report of how many documents were initially published by authors who don't belong to the document's primary organisation"
+  task :generate_csv, [:user_email] => :environment do |t, args|
+    require 'csv'
+
+    class CSVReport
+      def get_authors(edition)
+        edition.publication_audit_entry.edition.authors.first
+      end
+
+      def author_belongs_to_diff_org?(author, organisation)
+        author.organisation.present? && organisation.present? && organisation.id != author.organisation.id
+      end
+
+      def header_row
+        ["An admin link",
+         "Date first published",
+         "Title",
+         "Format",
+         "Lead organisation",
+         "First Author name",
+         "First Author email address",
+         "First Author organisation",
+         "Last Author name",
+         "Last Author email address",
+         "Last Author organisation"]
+      end
+
+      def get_last_author(edition)
+        edition.latest_edition.published_by
+      end
+
+      def filtered_editions
+        Edition.where("state = 'published' AND first_published_at >= '2014-01-01'")
```

Comment:
There's almost certainly a scope for published - I don't think it's gonna be more complicated than `state = 'published'` - but it might be.  In general it's better to use scopes that already exist and to speak in the language of the app, rather than the language of the framework.  Not least because if how we decide what is published changes in the future, we'll get that behaviour if we use the scope, but not if we write our own `where` clause.

It would be worth parameterising the date too (there might be a scope for it too, who knows).  Even though this is a "one-off" report for now it's the kind of thing that has a tendency to want to be run multiple times.